lib/rules/web/admin_console/4.8/common_ui_elements.xyaml

```diff
@@ -156,6 +156,9 @@ click_yaml_tab:
   params:
     tab_name: YAML
   action: click_tab
+click_yaml_tab_and_wait:
+  action: click_yaml_tab
+  action: wait_page_body_loaded
```

Comment:
This is not needed and we can merge it into action `check_encoded_secret`

Reply:
updated as suggested in the comment